packages/navigator/src/browser/navigator-frontend-module.ts

```diff
@@ -13,15 +13,23 @@ import { NavigatorMenuContribution } from './navigator-menu';
 import { FileNavigatorContribution } from './navigator-contribution';
 import { createFileNavigatorWidget } from "./navigator-container";
 import { WidgetFactory } from '@theia/core/lib/browser/widget-manager';
+import { CommandContribution } from '@theia/core/lib/common/command';
+import { KeybindingContribution } from '@theia/core/lib/common/keybinding';
 
 import '../../src/browser/style/index.css';
 
 export default new ContainerModule(bind => {
     bind(FileNavigatorContribution).toSelf().inSingletonScope();
     bind(FrontendApplicationContribution).toDynamicValue(c => c.container.get(FileNavigatorContribution));
-    bind(WidgetFactory).toDynamicValue(c => c.container.get(FileNavigatorContribution));
+    bind(CommandContribution).toDynamicValue(c => c.container.get(FileNavigatorContribution));
+    bind(KeybindingContribution).toDynamicValue(c => c.container.get(FileNavigatorContribution));
     bind(MenuContribution).to(NavigatorMenuContribution).inSingletonScope();
+
     bind(FileNavigatorWidget).toDynamicValue(ctx =>
```

Comment:
could you remove this binding, so it is not exposed and accidentally used?

Reply:
There are bindings for other widgets, too: `problem-frontend-module.ts`, `git-frontend-module.ts`, `extension-frontend-module.ts`. What about those?